src/cgame/cg_main.cpp

```diff
@@ -174,6 +174,7 @@ static Cvar::Cvar<float> cg_fov_level4("cg_fov_level4", "field of view (degrees)
 static Cvar::Cvar<float> cg_fov_human("cg_fov_human", "field of view (degrees) for humans", Cvar::NONE, 0);
 
 Cvar::Cvar<bool> ui_chatPromptColors("ui_chatPromptColors", "chat prompts (e.g. 'Say:') are color-coded", Cvar::NONE, true);
+Cvar::Cvar<std::string> ui_winner("ui_winner", "FOR INTERNAL USE", Cvar::NONE, "");
```

Comment:
Cvar::ROM and give proper description.

Reply:
We already use `FOR INTERNAL USE` with `cg_minimapActive`, the fact those are cvars is basically a HACK and we better implement a specific features for variables internal to the cgame and the ui.

Reply:
If this cvar is `Cvar::ROM` then `ui_winner` can never be updated and then when the game ends the cvar remains empty and the scoreboard doesn't display the winning team name.

Reply:
Are you sure? Cvar::ROM just means that users cannot modify it. The value can be changed with `Cvar::SetValue()`

Reply:
ROM, like LATCH, is not yet implemented for the gamelogic.

Reply:
illwieckz is right that this should not be a cvar anyway. We could make something like a data source but for singletons